default/stringtables/en.txt

```diff
@@ -1410,6 +1410,9 @@ Skip comparing the resource directory checksums.  This allows faster startup whe
 OPTIONS_DB_TESTING
 Linux only. AI will output log to console instead file for testing purpose.
 
+OPTIONS_DB_LOAD_OR_QUICKSTART
+Searches latest multi-player save game in autosave directory. Loads and starts it if found or starts new game. Requires hostless mode and no restriction on the minimum number of connected players, because the server will start with no connected players.
```

Comment:
maybe explain a bit about what quickstarting means for a hostless multiplayer game?

Reply:
I've tried to add more description.